web-local/src/lib/components/workspace/explore/view-containers/NoTimeSeriesContainer.svelte

```diff
@@ -0,0 +1,42 @@
+<section
+  class="grid items-stretch leaderboard-layout surface"
+  style:grid-template-columns="240px minmax(355px, auto)"
```

Comment:
Now that I see it, I'm a bit concerned that the difference between this and the time series version is just the first value in this `grid-template-columns`. It feels like this could easily be a prop passed in that defines the size of this particular element. This will also be useful down the line when we want to collapse this part of the grid further as a result of shrinking the container (not just the window, but the container itself).

I probably suggested having a separate container component for this case, but I think it's going to cause problems down the line w/ all the duplicated code.

What do you think?

Reply:
I think creating separate containers could be useful in cases where we want to deviate from the standard layout (such as in case of no dimensions we won't be needing the `leaderboard` section). Nevertheless, I have removed the new components for now. We can try this approach when the case arises